services/common/src/components/forms/RenderFileUpload.tsx

```diff
@@ -440,6 +442,13 @@ export const FileUpload = (props: FileUploadProps) => {
   const fileValidateTypeLabelExpectedTypesMap = invert(props.acceptedFileTypesMap);
   const acceptedFileTypes = uniq(Object.values(props.acceptedFileTypesMap));
 
+  const getLabel = (props) => {
+    if (props.labelHref)
+      return <a href={props.labelHref} target="_blank">{props.label}</a>
```

Comment:
Potential security risk here, needs a little bit of enhancement with the rel="noopener noreferrer" attribute.

Reply:
Thanks. This updated accordingly @henryoforeh-aot